tests/fixtures/jobs_info_no_stage.json

```
@@ -0,0 +1,24 @@
+{
+  "result": "ok",
+  "data": {
+    "ignore_conditions": ["free_space"],
+    "jobs": [
+      {
+        "name": "test_job_error_no_stage",
+        "reference": "test123",
+        "uuid": "3febe59311f94d6ba36f6f9f73357ca9",
+        "progress": 0,
+        "stage": null,
+        "done": true,
+        "errors": [
+          {
+            "type": "TestError",
+            "message": "Test error without stage field"
```

```suggestion
            "message": "Test error without stage field",
            "stage": None
```
Supervisor will never output an error without a stage on latest version.